### PR TITLE
[examples] Remove all uses of DefaultCollisionGroupManager

### PR DESCRIPTION
**Pull request by hugtalbot**
Simply remove all lines in scenes (scn) including a DefaultCollisionGroupManager

`$ find . -type f -name "*.scn" -exec sed -i '/DefaultCollisionGroupManager/d' {} +`

[ci-depends-on https://github.com/sofa-framework/Regression/pull/30]

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

File: modules/SofaMiscCollision/SofaMiscCollision_test/scenes/DefaultCollisionGroupManager_singleObject_test.scn [hunk 1]
```diff
@@ -25,7 +25,6 @@ response. The scene should work with or without the group manager.
     <BVHNarrowPhase/>
     <NewProximityIntersection name="Proximity" alarmDistance="2" contactDistance="0.7"/>
     <DefaultContactManager name="Response" response="PenalityContactForceField"/>
-    <DefaultCollisionGroupManager name="GroupManager"/>
```
Comment (alxbilger) [inline modules/SofaMiscCollision/SofaMiscCollision_test/scenes/DefaultCollisionGroupManager_singleObject_test.scn]: Probably not here

File: examples/Benchmark/Performance/CollisionGroupManager/1system_without_CollisionGroupManager.scn [hunk 2]
```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0"?>
 <!--
 This scene belongs to a collection of 3 scenes designed to evaluate the impact of
-the DefaultCollisionGroupManager component.
```
Comment (alxbilger) [inline examples/Benchmark/Performance/CollisionGroupManager/1system_without_CollisionGroupManager.scn]: Not in this file

File: examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn [hunk 3]
```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0"?>
 <!--
 This scene belongs to a collection of 3 scenes designed to evaluate the impact of
-the DefaultCollisionGroupManager component.
```
Comment (alxbilger) [inline examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn]: Not here

File: examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn [hunk 4]
```diff
@@ -1,15 +1,13 @@
...
 
 The 3 scenes are very similar: they all simulate two falling deformable cubes on a funnel.
 The collision prevents the cubes falling through the funnel.
 
 The difference between the 3 scenes is the way the mechanical system is solved.
 
 This scene solves two different systems, one per cube.
-There is a DefaultCollisionGroupManager component in this scene.
```
Comment (alxbilger) [inline examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn]: Not here

File: examples/Benchmark/Performance/CollisionGroupManager/2systems_without_CollisionGroupManager.scn [hunk 5]
```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0"?>
 <!--
 This scene belongs to a collection of 3 scenes designed to evaluate the impact of
-the DefaultCollisionGroupManager component.
```
Comment (alxbilger) [inline examples/Benchmark/Performance/CollisionGroupManager/2systems_without_CollisionGroupManager.scn]: Not here

File: examples/Benchmark/Performance/CollisionGroupManager/2systems_without_CollisionGroupManager.scn [hunk 6]
```diff
@@ -1,15 +1,13 @@
...
 
 The 3 scenes are very similar: they all simulate two falling deformable cubes on a funnel.
 The collision prevents the cubes falling through the funnel.
 
 The difference between the 3 scenes is the way the mechanical system is solved.
 
 This scene solves two different systems, one per cube.
-There is a DefaultCollisionGroupManager component in this scene.
```
Comment (alxbilger) [inline examples/Benchmark/Performance/CollisionGroupManager/2systems_without_CollisionGroupManager.scn]: Not here

**Review by alxbilger — CHANGES_REQUESTED** [review]
Some places where it must not be removed

**Comment by fredroy** [comment]
[ci-build][with-all-tests]

**Comment by fredroy** [comment]
@hugtalbot you will need to update the regression file for collisionMultiple.scn, as the DefaultCollisionGroupManager seemed to have an impact in this one

**Comment by fredroy** [comment]
[ci-build][force-full-build][with-all-tests]

**Review by fredroy — APPROVED** [review]

**Pull request merged**